### ISSUE TITLE
Bug with custom config saving?

### ISSUE THREAD

**Issue by ThePoleThatFishes** [ThizThizzyDizzy/nc-reactor-generator#141]
I'm trying to make an ncpf for my modpack, as it includes QMD, a modified version of MRF, some extra coils (the latter two bundled into a custom addon) and modified fuel values. I import the nuclearcraft.cfg, add QMD and the addon ncpf, save all of it as a full config ncpf, all is well. Then I load the ncpf and the heatsink rules are all kinds of fucked up, eg. obsidian requires an axial pair... of itself!?

Attaching the cfg, the addon ncpf and the final ncpf.
[configs.zip](https://github.com/ThizThizzyDizzy/nc-reactor-generator/files/12285239/configs.zip)

**Comment by ThizThizzyDizzy** [comment]
The spaghetti of legacy NCPF falling apart at the seams...

The only save/load issue I'm aware of is when you have an addon that uses blocks from another addon in placement rules
I also haven't checked turbines as thoroughly though

Most likely won't be fixed though, the planner is in the middle of a major refactor (including a NCPF rewrite that will prevent this)

**Comment by ThizThizzyDizzy** [comment]
Hit the wrong button x.x

**Comment by ThizThizzyDizzy** [comment]
huh, after some testing, this appears to be an odd issue surrounding the .cfg import?
exploring the .cfg (after save/load) also shows the obsidian rules mismatched, while loading the addon into the base config still ends up fine

**Comment by Gofer2x** [comment]
Experienced this too and was going to make an issue, but I guess no point now.

Though another, possibly related thing - the pack I'm playing, Enigmatica 2 Expert Extended has a custom config, and it seems the planner doesn't actually recognize/consider many(?) of the config options properly. Many of the values are not present in the GUI, but also seem to not reflect in-game behavior either.
![image](https://github.com/ThizThizzyDizzy/nc-reactor-generator/assets/45763255/a5c3bcec-ee0a-4927-aa3c-71d44dd8e644)
I noticed this when my in-game reactor had x6 as much efficiency and steam production as what was shown in the planner (with the E2EE .cfg loaded). Ditto with the fuel times (modified by the config) also still showing the default values when they should be halved.
Basically, configs are all kinds of messed up it seems.

The rewrite could make all this irrelevant possibly, but thought I should still point it out.

**Comment by ThizThizzyDizzy** [comment]
The multipliers aren't in the GUI as they are applied to the fuel stats on import;
However, I probably forgot to make it use the overhaul multipliers since there were no modpacks that used them at the time
